core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMaterializedViewDefinition.java

```diff
@@ -33,6 +34,16 @@
     private final Optional<String> comment;
     private final Optional<String> owner;
     private final Map<String, Object> properties;
+    /**
+     * Set of versioning columns that engine will provide during MV refresh.
+     * MV storage table should provide same set of columns with matching names.
+     * Empty if MV does not support versioning.
+     */
+    private final Optional<Set<Column>> versioningColumns;
+    /**
+     * Current MV version. Empty if MV does not support versioning.
+     */
+    private final Optional<TableVersion> version;
```

Comment:
Why is it part of `ConnectorMaterializedViewDefinition`? This is dynamically changing information, and it should not be part of this class.